Client/WebFilters/FilterListResourceDownloader.swift

```diff
@@ -157,6 +167,22 @@ public class FilterListResourceDownloader: ObservableObject {
     _ = await (cachedFilterLists, cachedDefaultFilterList)
   }
   
+  /// Enables a filter list for the given uuid. Returns true if the filter list exists or not.
+  public func enableFilterList(forFilterListUUID uuid: String, isEnabled: Bool) -> Bool {
+    // Enable the setting
+    if let index = filterLists.firstIndex(where: { $0.uuid == Self.cookieConsentNoticesUUID }) {
```

Comment:
nit: I believe we dont need to fetch the index first and get the value using the index and just use r

`if var filterList = filterLists.first(where: { $0.uuid == Self.cookieConsentNoticesUUID }) `

Reply:
oh yeah you are totally right, I though you are setting the modified value somewhere.